src/runtime_init_memory.js

```diff
@@ -30,7 +30,11 @@ function initMemory() {
   } else
 #endif
   {
+#if MINIMAL_RUNTIME
+    var INITIAL_MEMORY = {{{ INITIAL_MEMORY }}};
+#else
     var INITIAL_MEMORY = {{{ makeModuleReceiveExpr('INITIAL_MEMORY', INITIAL_MEMORY) }}};
```

Comment:
The `makeModuleReceiveExpr` macro already takes care of this I think.  

IIRC, `MINIMAL_RUNTIME` sets `INCOMING_MODULE_JS_API` to empty by default which should make this expand to just `INITIAL_MEMORY`